Prace_domowe/Praca_domowa2/Chylak_Maciej/Chylak_Maciej_PD2.ipynb

```diff
@@ -0,0 +1,2914 @@
+{
```

Comment:
<p>Błąd należałoby wyliczyć tylko dla wartości imputowanych, bo gdy wyliczamy dla całej kolumny wynik jest zakłamany, ponieważ we wzorze jest wyliczana średnia, przez co liczność ma znaczenie.</p><p> </p><p><u>Prośba o zmodyfikowanie sposobu wyliczania błędu.</u></p><p> </p><p>Może lepiej byłoby operować na pierwiastku z błędu średniokwadratowego.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>